public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -23,31 +24,33 @@ export function AppChrome({ children }: Props) {
     return <main className="main-view">{children}</main>;
   }
 
+  const chromeless = state.chromeless || state.kioskMode === KioskMode.Full;
+  const searchBarHidden = state.searchBarHidden || state.kioskMode === KioskMode.TV;
```

Comment:
should we just update the `chromeless`/`searchBarHidden` state in `AppChromeService` when we update `kioskMode`, then we wouldn't need these `||`s?

Reply:
But then we would loose kiosk mode when moving between routes

Reply:
ahhhhhhh got it 👍